MelonLoader.Installer/Views/DetailsView.axaml.cs

```diff
@@ -138,12 +138,27 @@ private void InstallHandler(object sender, RoutedEventArgs args)
         Model.Installing = true;
         ShowLinuxInstructions.IsVisible = false;
 
-        _ = MLManager.InstallAsync(Path.GetDirectoryName(Model.Game.Path)!, Model.Game.MLInstalled && !KeepFilesCheck.IsChecked!.Value,
+        _ = MLManager.InstallAsync(Path.GetDirectoryName(Model.Game.Path)!, Model.Game.Id, Model.Game.MLInstalled && !KeepFilesCheck.IsChecked!.Value,
             (MLVersion)VersionCombobox.SelectedItem!, Model.Game.IsLinux, Model.Game.Is32Bit,
             (progress, newStatus) => Dispatcher.UIThread.Post(() => OnInstallProgress(progress, newStatus)),
             (errorMessage) => Dispatcher.UIThread.Post(() => OnOperationFinished(errorMessage)));
     }
 
+    private void GamePropsHandler(object sender, RoutedEventArgs args)
```

Comment:
Keep in mind that the installer supports multiple launchers. Make sure you only show this on Steam games or add support for other games